src/main/java/org/spongepowered/api/entity/living/animal/HorseStyle.java

```diff
@@ -33,13 +34,6 @@
  * horse. The style can be inherited to new born child horses.</p>
  */
 @CatalogedBy(HorseStyles.class)
-public interface HorseStyle extends DataSerializable {
-
-    /**
-     * Gets the name of this style.
-     *
-     * @return The name of this style
-     */
-    String getName();
+public interface HorseStyle extends CatalogType {
```

Comment:
Is the removal of `extends DataSerializable` intentional?
Because the import has not been removed.

This also has happened in some other classes.